hd-docs/backend/collector-service.md

```
@@ -0,0 +1,364 @@
+  }
+  
+  // Generate document with OCR'd text
+  const data = {
+    id: v4(),
+    pageContent: content,
+    token_count_estimate: tokenizeString(content),
+    // ... metadata
+  };
+  
+  return { success: true, documents: [document] };
+}
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)